src/quo2/components/drawers/drawer_top/style.cljs

```diff
@@ -0,0 +1,27 @@
+(ns quo2.components.drawers.drawer-top.style
+  (:require [quo2.foundations.colors :as colors]))
+
+(def container
+  {:padding-horizontal 20
+   :padding-bottom     12
+   :flex-direction     :row})
+
+(def body-container
+  {:flex 1})
+
+(defn description
+  [theme blur?]
+  {:color (if blur?
+            colors/white-opa-40
+            (colors/theme-colors colors/neutral-50 colors/neutral-40 theme))})
+
+(def left-container
+  {:margin-right    8
+   :justify-content :center})
+
+(defn network-text-color
+  [network]
+  {:color (get colors/networks network)})
```

Comment:
hmm, I think we should use the custom-color resolving here. Same as we are using for everything else. 
Can we add this map to the custom-colors function and then use as we are doing in the rest. That way we abstract these types of colors to that mechanism and then can just adjust that at a later as needs be 👍

Reply:
i.e imo we should update `colors` map in the colors file to 
```
(def colors-map
  (merge {:primary {50 primary-50 ;; User can also use primary color as customisation color
                    60 primary-60}
          :beige   {50 "#CAAE93"
                    60 "#AA927C"}
         ...
         customization
         networks))
  ```

Reply:
so we can use `(custom-colors network 50)` etc 👍

Reply:
perhaps we make these maps like networks private too using `def-`

Reply:
Done